### PR TITLE
Start to move Firebase Parts to MVC structure

### PR DESCRIPTION
**Pull request by krazhopper**
#32 

This was just a moving of the existing code to our giving structure. We need to discuss this further on how we can restructure the code itself to get a more MVC style structured code.

### CODE REVIEW

**Comment by krazhopper** [comment]
close #32

**Comment by DavidMatalik** [comment]
> #32
> 
> This was just a moving of the existing code to our giving structure. We need to discuss this further on how we can restructure the code itself to get a more MVC style structured code.

That means, I don't have to put effort right now in understanding the code, right? Or is there something I should specifically look at?

File: src/model/userdbmapper.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+const deleteAccount = function(firebase, database) 
+{
+   var user = firebase.auth().currentUser;
+   // delete user in database
+   remove(ref(database, 'users/' + user.uid)).then(function(error)
+   {
+       if (error)
+       {
+           console.log(error.message)
+       }
```
Comment (DavidMatalik) [inline src/model/userdbmapper.js:34]: Syntax-Aussehen ist hier anders als in anderen files, z.B in checkTitle in itemmodel.
```
if (title.length > 255) {
      throw new Error('too many characters')
    }
```
D.h. es sollte so geschrieben werden: 
```
if (error) {
  console.log(error.message)
}
```
Und das ist auch an anderen Stellen. Vermutlich wäre es am Besten/Einfachsten an dieser Stelle prettier in unser Projekt einzuführen. Dann hätten wir ein einheitliches Aussehen vom code. Was meinst du? Wir könnten den PR dann mal so mergen und ich könnte in einem PR danach die index.js und prettier reinbringen.

**Review by DavidMatalik — CHANGES_REQUESTED** [review]
Additional to the one comment: as said in our conversation yesterday. Before merging, all var's should be replaced by either const or let, depending on the situation.

File: src/model/userdbmapper.js (hunk 1 above)
Comment (krazhopper) [reply src/model/userdbmapper.js:34]: Yes, I agree. There is a mix of different error handling. If you would like to introduce prettier, feel free to do so in your new branch ;)

**Review by DavidMatalik — APPROVED** [review]

**Pull request merged**